tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,298 @@
+"""Tests for leapp upgrade of content hosts with Satellite
+
+:Requirement: leapp
+
+:CaseLevel: Integration
+
+:CaseComponent: LeappIntegration
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from wait_for import wait_for
+
+from robottelo.cli.repository import Repository
+from robottelo.constants import DEFAULT_ARCHITECTURE
+from robottelo.constants import PRDS
+
+global gbl_var_ak
+RHEL_REPOS = {
+    'rhel7_9_server': {
+        'id': 'rhel-7-server-rpms',
+        'name': 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7.9',
+        'releasever': '7.9',
+        'reposet': 'Red Hat Enterprise Linux 7 Server (RPMs)',
+        'product': 'Red Hat Enterprise Linux Server',
+    },
+    'rhel7_server_extras': {
+        'id': 'rhel-7-server-extras-rpms',
+        'name': 'Red Hat Enterprise Linux 7 Server - Extras RPMs x86_64',
+        'releasever': '7',
+        'reposet': 'Red Hat Enterprise Linux 7 Server - Extras (RPMs)',
+        'product': 'Red Hat Enterprise Linux Server',
+    },
+    'rhel8_8_bos': {
```

Comment:
This is moving target. 
Once 8.10/9.4 is released it is going to fail.
We will be changing this from time to time.
So please create a constansts in this module:
```
RHEL7_VER = '7.9'
RHEL8_VER = '8.8'
RHEL9_VER = '9.2'
```
then by changing single constant you accommodate test for future versions

Ofc get rid of minor RHEL versions out of keys in `RHEL_REPOS` dictionary (it is useless there):

`rhel8_8_bos:`  -> `rhel8_bos:`
```
'rhel8_8_bos': {
    ...
    'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS RPMs {RHEL8_VER}',
    'releasever': RHEL8_VER
'rhel8_aps': {
    ...
}

Reply:
I believe that Lukas means something like this:
https://github.com/ogajduse/robottelo/commit/d937ec7141a734885befb4a2805b3709e3cb653c